csrankings-m.csv

```diff
@@ -321,6 +321,7 @@ Manuel V. Hermenegildo,IMDEA Software Institute,http://software.imdea.org/people
 Manuel Wimmer,JKU Linz,https://www.se.jku.at/manuel-wimmer,YZDY1psAAAAJ
 Manuel Álvarez Díaz,University of A Coruña,https://pdi.udc.es/es/File/Pdi/WJ6AF,aqGNCqIAAAAJ
 Manya Ghobadi,Massachusetts Institute of Technology,http://people.csail.mit.edu/ghobadi,M78g4SAAAAAJ
+Manyi Li,Shandong University,https://manyili12345.github.io/,U_YYKJkAAAAJ

```

Comment:
Says "Associate Researcher", not clearly tenure-track.

Reply:
We deleted Li Manyi's information.